vendor/bat-native-ads/src/bat/ads/internal/ads_history/ads_history.cc

```diff
@@ -88,5 +89,13 @@ void AddInlineContentAd(const InlineContentAdInfo& ad,
   Client::Get()->AppendAdHistory(ad_history);
 }
 
+void AddSearchResultAd(const SearchResultAdInfo& ad,
```

Comment:
why are we adding new functions with similar functionality? should we just use one function, i.e. `AddAdHistory`

Reply:
I have a list of tech debt issues to remove duplicate code, I will triage this soon